timemachine/cpp/src/local_md_potentials.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+#include "device_buffer.hpp"
+#include "pinned_host_buffer.hpp"
+#include <memory>
+#include <vector>
+
+#include "bound_potential.hpp"
+#include "curand.h"
+#include "flat_bottom_bond.hpp"
+#include "local_md_utils.hpp"
+
+namespace timemachine {
+
+class LocalMDPotentials {
+
+public:
+    LocalMDPotentials(const int N, const std::vector<std::shared_ptr<BoundPotential>> bps);
```

Comment:
why not `const ... &bps`?

Reply:
Done in https://github.com/proteneer/timemachine/pull/1004/commits/ff9ded78bfea7a3cfc9f7a74bba3f1b8992ddb99